Sources/GXUITest/GXUITestingAPI.swift

```diff
@@ -874,6 +874,7 @@ fileprivate let _tapableElementTypes: Array<XCUIElement.ElementType> = [.button,
 																		.checkBox,
 																		.switch,
 																		.segmentedControl,
+                                                                        .scrollView,
```

Comment:
`.scrollView` element type was added to `_tapableElementTypes`, but it's not present in `_allElementTypes`.
Also, use tabs instead of spaces to match indentation style.

Reply:
FYI: for the next time, there is a button to re-request review at the top.
<img width="408" alt="image" src="https://github.com/user-attachments/assets/98476927-c330-48af-9c10-b56ba2d8b3dd" />